.github/workflows/deployment.yml

```diff
@@ -163,6 +163,19 @@ jobs:
             rm ${{ secrets.SSH_PATH_UPDATES }}/${{ needs.build-package.outputs.packageUploadVersion }}/package.tar.gz
             chmod 777 --recursive ${{ secrets.SSH_PATH_UPDATES }}/${{ needs.build-package.outputs.packageUploadVersion }}/*
 
+      # Upload the installer executable to the server
+      - name: Upload installer
+        uses: appleboy/scp-action@master
```

Comment:
Isn't scp available in windows runner in powershell? It is not a good idea provide ip, login, password for non-github actions.